tests/plugins/precision/test_deepspeed_precision.py

```diff
@@ -0,0 +1,32 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+from pytorch_lightning.plugins.precision.deepspeed import DeepSpeedPrecisionPlugin
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from pytorch_lightning.utilities.imports import _DEEPSPEED_GREATER_EQUAL_0_6
+from tests.helpers.runif import RunIf
+
+
+@RunIf(deepspeed=True)
+def test_invalid_precision_with_deepspeed_precision():
+    with pytest.raises(ValueError, match="is not supported. `precision` must be one of"):
+        DeepSpeedPrecisionPlugin(precision=64, amp_type="native")
+
+
+@RunIf(deepspeed=True)
+@pytest.mark.skipif(_DEEPSPEED_GREATER_EQUAL_0_6, reason="requires deepspeed < 0.6")
+def test_incompatible_bfloat16_raises_error_with_deepspeed_version():
```

Comment:
This is never run by CI right? I get the point of having it, but just confirming our CI will never run this so the code isn't covered? Might be worth mocking or something

Reply:
done!